test/Date/DateUTC-DateGMT-same.js

```diff
@@ -0,0 +1,11 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+const areSame = Date.prototype.toGMTString === Date.prototype.toUTCString;

```

Comment:
I thought loading the UnitTestFramework and writing the boilerplate for the framework for this one tiny test case was overkill.